### PR TITLE
Update doc strings for command reference

### PR DESCRIPTION
**Pull request by jbogarthyde**
This is an initial draft transferring content from google doc draft to JSON sources. 
Basic command descriptions and generation schematic options included.
This is to meant to test the git and cli-doc build process as well.

### CODE REVIEW

File: packages/angular/cli/commands/doc.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/doc.json",
-  "description": "Opens the official Angular API documentation for a given keyword.",
-  "$longDescription": "",
+  "description": "Opens the Angular documentation in a browser, with a search for a given term.",
+  "$longDescription": "./doc.md,
```
Comment (clydin) [inline packages/angular/cli/commands/doc.json]: missing end quote

File: packages/angular/cli/commands/generate.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/generate.json",
-  "description": "Generates and/or modifies files based on a schematic.",
-  "$longDescription": "",
+  "description": "Creates, initializes, and modifies files to create a generic version of an app or app element in the given or default project, according to the given schematic.",
+  "$longDescription": "Takes the name of the project, as specified in the \"projects\" section of the angular.json workspace configuration file. \n When a project name is not supplied, the configured default project of the workspace is used.",
```
Comment (clydin) [inline packages/angular/cli/commands/generate.json]: The generate command actually doesn't take the name of a project in the same way as the other commands.  It is currently an outlier and we've been discussing the best path forward to potentially remedy the situation.

Comment (clydin) [reply packages/angular/cli/commands/generate.json]: `@longDescription` also needs to point to a separate file.

File: packages/angular/cli/commands/doc.json (hunk 1 above)
Comment (jbogarthyde) [reply packages/angular/cli/commands/doc.json]: moved long desc back into short desc

File: packages/angular/cli/commands/generate.json (hunk 2 above)
Comment (jbogarthyde) [reply packages/angular/cli/commands/generate.json]: No long description now. 
I'm also trying to formulate better boilerplate for the ones that do take an optional project. Maybe we can add it to definitions.json

**Pull request merged**